staging/src/k8s.io/sample-apiserver/pkg/admission/plugin/banflunder/admission_test.go

```diff
@@ -56,7 +56,7 @@ func TestBanflunderAdmissionPlugin(t *testing.T) {
 					Namespace: "",
 				},
 			},
-			admissionInputKind:     wardle.Kind("Flunder").WithVersion("version"),
```

Comment:
these changes shouldn't be necessary

Reply:
when using the internal API structure to execute the test will panic
```
/usr/local/go/src/runtime/asm_amd64.s:1333
panic: interface conversion: runtime.Object is *wardle.FischerList, not *v1alpha1.FischerList [recovered]
	panic: interface conversion: runtime.Object is *wardle.FischerList, not *v1alpha1.FischerList

goroutine 26 [running]:
k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/Users/admin/go/src/k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:58 +0x108
panic(0x1b960c0, 0xc0001c34a0)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/client/clientset/versioned/typed/wardle/v1alpha1/fake.(*FakeFischers).List(0xc0000c0398, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/admin/go/src/k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/client/clientset/versioned/typed/wardle/v1alpha1/fake/fake_fischer.go:62 +0x5f9
k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/client/informers/externalversions/wardle/v1alpha1.NewFilteredFischerInformer.func1(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1ca3ef7, ...)
	/Users/admin/go/src/k8s.io/kubernetes/vendor/k8s.io/sample-apiserver/pkg/client/informers/externalversions/wardle/v1alpha1/fischer.go:63 +0x126
```

Reply:
I was referring specifically to the groupversionkind produced by this line... the old and new lines should have identical output

Reply:
No Kind() method in versioned/register.go. so i add this line